eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -2126,6 +2126,66 @@ public function deleteTranslationFromDraft(APIVersionInfo $versionInfo, $languag
         }
     }
 
+    /**
+     * Hides Content by making all the Locations appear hidden.
+     * It does not persist hidden state on Location object itself.
+     *
+     * Content hidden by this API can be revealed by revealContent API.
+     *
+     * @see revealContent
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
+     */
+    public function hideContent(ContentInfo $contentInfo): void
+    {
+        if (!$this->repository->canUser('content', 'edit', $contentInfo)) {
```

Comment:
don't we have a content/hide policy?

Reply:
> don't we have a content/hide policy?

Hmm, good point. Indeed we use `content/hide` to manage Locations. @SylvainGuittard should we use the same permission as in case of hiding single Location, or `content/edit` is expected to work for this feature?

Reply:
Totally! `content/hide` should be used here.
@alongosz we have `content/manage locations` to manage locations (add and remove)